### DISCUSSION TITLE
Возможность указывать индивидуальное нерабочее время по входящим маршрутам

### DISCUSSION THREAD

**Discussion by DanilRomanov1989** [mikopbx/Core#83] [Ideas (Идеи)]
От версии к версии меняется метод окостыливания для реализации данного функционала. Предлагаю сделать к нему GUI, думаю многим будет актуально.

**Comment by boffart** [comment]
Еще ссылка на обсуждение 
https://qa.mikopbx.com/9984/режим-нерабочего-времени
